### PR TITLE
refactor: update topic label style for article page

### PR DESCRIPTION
**Pull request by taylrj**
Address [TWREPORTER-158](https://twreporter-org.atlassian.net/browse/TWREPORTER-158).

This patch updates topic label style for article page to increase margin
between header and topic label for normal article page.

### CODE REVIEW

File: packages/react-article-components/src/components/leading/normal.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ const Topic = styled.div`
   border-width: 1px;
   border-style: solid;
   padding: 10px;
-  margin-bottom: 15px;
+  margin: 20px 0 15px 0;
```
Comment (Aylie-Chou) [inline packages/react-article-components/src/components/leading/normal.js:73]: It seems not all article has topic part.
How about modify padding-top in L106 ?

Comment (taylrj) [reply packages/react-article-components/src/components/leading/normal.js]: Updated.

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Comment by taylrj** [comment]
Clean rebase.

**Pull request merged**